public/js/apps/transaction/app.jsx

```diff
@@ -37,6 +37,8 @@ export default class TransactionApp extends Component {
     this.state = {
       accessToken: qs.access_token,
       productId: qs.product,
+      // This is an amount to pay, which only applies to donations.
```

Comment:
I predict that comment will rot, its possible we'll have a "pay what you want" product in the future. Don't want to hang too many things off of one settings value...